test/Participant.cpp

```diff
@@ -237,6 +237,11 @@ void Participant::setMeshTetrahedra
     precice::span<const precice::VertexID> vertices)
 {}
 
+void Participant::resetMesh
+(
+    precice::string_view meshName)
+{}
```

Comment:
If such a mock function exists, can you please add a unit test?

Reply:
I added a test without assertions. Is this fine?

Reply:
Looks good :+1: